shs/update_td_b_up.sh

```
@@ -0,0 +1,23 @@
+# cd /data/project/mdwiki/
+
+rm -rf tdx
+
+# Download the wd-core repository from GitHub.
+git clone -b update https://github.com/MrIbrahem/Translation-Dashboard.git tdx
+
+# delete composer.json and composer.lock
+rm -rf tdx/composer.json tdx/composer.lock
+
+# delete all json files in all subdirectories
+find tdx -name *.json -delete
```

Shellcheck (0.10.0)

[warning] Quote the parameter to -name so the shell won't interpret it.

(SC2061)